website/docs/configurations/02-gpu-operator.md

```diff
@@ -2,55 +2,47 @@
 title: GPU Operator
 ---
 
+This guide details recommended configurations for GPU Operator v24.9.2 to enable GPU workload, with specific settings for GPUDirect RDMA integration.
+
 :::tip
 This guide assumes a basic understanding of GPU Operator and its role in Kubernetes clusters. Readers unfamiliar with GPU Operator are advised to review the official [guide](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/index.html) before proceeding. The concepts and recommended configurations presented here build on that foundation to enable GPU workload and GPUDirect RDMA in AKS. This documentation is based on GPU Operator v24.9.2.
 :::
 
-## Recommended Configuration
+## GPU Drivers: AKS-managed vs. GPU Operator-managed
 
-This guide details recommended configurations for GPU Operator v24.9.2 to enable GPU workload, with specific settings for GPUDirect RDMA integration.
+:::danger
+AKS-managed GPU drivers and the NVIDIA GPU Operator managed GPU drivers are **mutually exclusive** and cannot coexist. When you create a nodepool **without** the `--skip-gpu-driver-install` flag, AKS provisions it with a node image that includes pre-installed NVIDIA drivers and the NVIDIA container runtime. Installing GPU Operator subsequently replaces this setup by deploying its own `nvidia-container-toolkit`, overriding the AKS-managed configuration. Upon uninstalling GPU Operator, the toolkit cannot revert to the original AKS containerd configuration, as it lacks awareness of the prior state, potentially disrupting the node’s container runtime and impairing workload execution.
+:::
 
-### Skip GPU Driver Installation
+When provisioning GPU nodepools in an AKS cluster, the cluster administrator has the option to either rely on the default GPU driver installation managed by AKS or via GPU Operator. This decision impacts cluster setup, maintenance, and compatibility.
+
+|                | **AKS-managed GPU Driver (Without GPU Operator)**                                  | **GPU Operator-managed GPU Driver (`--skip-gpu-driver-install`)**                         |                                                                 |
+|----------------|------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|-----------------------------------------------------------------|
+| **Automation** | AKS-managed drivers; cluster administrator needs to manually deploy device plugins | Automates installation of driver, device plugins, and container runtimes via GPU Operator |                                                                 |
+| **Complexity** | Simple, no additional components except device plugins                             | More complex, requires GPU Operator and additional components                             | Moderate setup (deploy operator); simplified ongoing management |
```

Comment:
The markdown table format was not rendered correctly
![image](https://github.com/user-attachments/assets/559bcb68-08d9-4099-b9af-119586950960)